pkg/vipertools/vipertools.go

```diff
@@ -60,3 +60,18 @@ func FirstNonEmptyString(v *viper.Viper, keys ...string) (string, bool) {
 func GetString(v *viper.Viper, key string) string {
 	return strings.Trim(v.GetString(key), `"'`)
 }
+
+// GetStringMapString gets a parameter/setting by key prefix and stripts any quotes.
+func GetStringMapString(v *viper.Viper, prefix string) map[string]string {
+	m := map[string]string{}
+
+	for _, k := range v.AllKeys() {
+		if !strings.HasPrefix(k, prefix) {
```

Comment:
This should be `if !strings.HasPrefix(k, prefix + ".") {` to prevent matching sections starting with prefix, for ex:

`[project_api_key_foo]`